src/Http/Middleware/RouteStatisticsMiddleware.php

```diff
@@ -12,7 +12,6 @@ class RouteStatisticsMiddleware
      * Handle an incoming request.
      *
      * @param  \Illuminate\Http\Request  $request
-     * @param  \Closure  $next
```

Comment:
This should not be deleted in this PR

Reply:
I can add it again, it is however discouraged to strong type function arguments and add it to the @phpdoc entries.

Reply:
This PR should only toutch on the files/lines relevant for this PR. If you have suggestion for other improvements, feel free to submit a second PR. But the number of lines affected by this PR should be at a minimum.